### PR TITLE
Added more tests for UploadedFiles

### PR DESCRIPTION
**Pull request by dopesong**

### CODE REVIEW

File: tests/Http/UploadedFilesTest.php [hunk 1]
```diff
@@ -185,6 +276,45 @@ public function providerCreateFromEnvironment()
                     ],
                 ]
             ],
+            // array of files as multidimensional array: <input name="avatars[]" type="file">
+            [
+                // $_FILES array
```
Comment (akrabat) [inline tests/Http/UploadedFilesTest.php:281]: $_FILES doesn't look like this at all

Comment (akrabat) [inline tests/Http/UploadedFilesTest.php]: http://php.net/manual/en/features.file-upload.multiple.php explains the format when you have two files uploaded with the same name as an array.

Comment (akrabat) [inline tests/Http/UploadedFilesTest.php:281]: Ignore me - I'm thinking of the case where the name is `foo[bar][]`… see https://github.com/php-fig/fig-standards/pull/746/files

Comment (dopesong) [inline tests/Http/UploadedFilesTest.php:281]: /ignore @akrabat

**Pull request merged**